boot/freeldr/freeldr/arch/i386/entry.S

```diff
@@ -73,27 +74,25 @@ _FrldrStartup:
     call _InitIdt
 
 #ifndef _USE_ML
-    /* Clean out bss */
+    /* Clean out BSS -- Only for GCC build since MSVC one has it merged with TEXT (see CMakeLists.txt) */
```

Comment:
Recalling the reader why.

Reply:
But it will most likely reduce the virtual size.